config/locales/client.en.yml

```diff
@@ -1996,9 +1996,11 @@ en:
       unread: "unread"
       new_topics:
         one: "%{count} new topic"
+        few: "%{count} new topics"
```

Comment:
You can't add the `few` key to pluralized strings in English. English uses only `one` and `other`.